src/components/form-intro.js

```diff
@@ -0,0 +1,83 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import {useForm} from 'react-hook-form';
+
+console.clear();
+
+export default function IntroForm() {
+	const {
+		register,
+		handleSubmit,
+		formState: {errors},
+	} = useForm();
+
+	const [people, setPeople] = useState([
+		{
+			restaurantName: 'Restaurant Name',
+			typeOfFood: 'Type of Food',
+			priceLevel: 'Price Level',
+			id: nanoid(),
+		},
+	]);
+
+	function onSubmit(data) {
+		console.log(data);
+
+		setPeople([...people, {...data, id: nanoid()}]);
+	}
+
+	return (
+		<>
+			<form onSubmit={handleSubmit(onSubmit)}>
+				<input
+					type="text"
+					placeholder="Name of Restaurant"
+					id="restaurantName"
+					{...register('restaurantName', {
+						required: true,
+					})}
+				/>
```

Comment:
This input is missing a label and therefore is not really accessible. Please add a label 🙂